libafl_libfuzzer/README.md

```diff
@@ -51,6 +51,22 @@ As this branch generally offers the highest performance version of `libafl_libfu
 Remember to `cargo update` often if using the experimental changes, and please [submit an issue]
 if you encounter problems while using `libfuzzer-best`!
 
+#### macOS
+
+On macOS, you will need to add weak linking for some functions in a `build.rs` file:
```

Comment:
Can't we add this into our own libafl_libfuzzer crate's build.rs file with the same result?

Reply:
Unfortunately no, I tried this but found out it's disallowed:  https://github.com/rust-lang/cargo/issues/9554

Reply:
It's possible there's a better way to do this. I'm not sure how libfuzzer-sys makes this work

Reply:
Poked around and it is doing some `nullptr` stuff with C++: https://github.com/rust-fuzz/libfuzzer/blob/9b8c0a7a66fdb5a8fcb627e64428c725795082c3/libfuzzer/FuzzerExtFunctionsWeak.cpp#L44